src/config/profile.h

```diff
@@ -261,6 +261,78 @@ typedef struct {
   ARRAY_MEMBER(elements, OSD_NUMBER_ELEMENTS, uint32) \
   ARRAY_MEMBER(elements_hd, OSD_NUMBER_ELEMENTS, uint32)
 
+typedef struct {
+  uint32_t led_mask1;
+  uint32_t led_mask2;
+  uint16_t colors[3];
+} rgb_map_t;
+
+#define RGB_MAP_MEMBERS            \
+  MEMBER(led_mask1, uint32)        \
+  MEMBER(led_mask2, uint32)        \
+  ARRAY_MEMBER(colors, 3, uint16)  \
+
+typedef struct {
+  rgb_map_t led_map[100];
+  uint8_t num_steps;
+  uint32_t duration;
+  uint8_t pattern_reverse;
+} rgb_sequence_t;
+
+#define RGB_SEQ_MEMBERS                  \
+  ARRAY_MEMBER(led_map, 100, rgb_map_t)  \
+  MEMBER(num_steps, uint8)               \
+  MEMBER(duration, uint32)               \
+  MEMBER(pattern_reverse, uint8)         \
```

Comment:
trailing `\` with this one and all the following. likely this is messing up the code generation.